hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -1,22 +1,36 @@
 # USB mass storage boot
 
-**Available on Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, and 4B only.**
+**Available on Raspberry Pi 2B v1.2, 3A+, 3B, 3B+, 4B, 400 and Compute Module 4 only.**
 
 This page explains how to boot your Raspberry Pi from a USB mass storage device such as a flash drive or a USB hard disk. When attaching USB devices, particularly hard disks and SSDs, be mindful of their power requirements. If you wish to attach more than one SSD or hard disk to the Pi, this normally requires external power - either a powered hard disk enclosure, or a powered USB hub. Note that models prior to the Pi 4 have known issues which prevent booting with some USB devices.
 
-See the [bootmodes documentation](README.md) for the boot sequence and alternative boot modes (network, USB device, GPIO or SD boot).
-
-Note that 'USB mass storage boot' is different from 'USB device boot mode'. [USB device boot mode](device.md) allows a Raspberry Pi connected to a computer to boot as a USB device, using files from that computer.
-
-If you are unable to use a particular USB device to boot your Raspberry Pi, an alternative is to use the special bootcode.bin-only boot mode as described [here](README.md). This Pi will still boot from the SD card, but `bootcode.bin` is the only file read from it.
+<a name="pi400"></a>
+## Raspberry Pi 400
+To boot the Pi 400 from a USB mass storage device, simply image the USB drive with Raspberry Pi OS 2020-08-20 or newer using the Raspberry Pi Imager utility: select the USB drive from the `SD Card` list in Raspberry Pi Imager.
 
 <a name="pi4"></a>
 ## Raspberry Pi 4
-To enable USB mass storage boot on a Raspberry Pi 4:
+Depending on when your Raspberry Pi 4 was manufactured, the bootloader EEPROM may need to be updated to enable booting from USB mass storage devices. 
+
+### Check if your Pi 4B has the required bootloader EEPROM version
+
+To check if your Pi 4B has the required bootloader EEPROM version, boot the device with no SD card inserted and a display attached to one of the HDMI ports. The Pi 4B will display a diagnostic screen on the attached display, which includes the bootloader EEPROM version at the top of the screen. The bootloader must be dated `Sep 3 2020` or later to support USB mass storage boot. If the diagnostic screen reports a date earlier than `Sep 3 2020`, or there is no diagnostic screen shown, you will need to updated the bootloader EEPROM first to enable USB mass storage boot.
```

Comment:
Maybe change "boot the device" to "power up the device", since nothing is actually "booting"? :shrug: 

"you will need to updated" -> "you will need to update"

Reply:
fixed